databricks.yml

```
@@ -0,0 +1,94 @@
+            node_type_id: "Standard_DS3_v2"
+            driver_node_type_id: "Standard_DS3_v2"
+            autoscale: 
+              min_workers: 1
+              max_workers: 1
+        
+      tasks:
+        - task_key: "preprocessing"
+          job_cluster_key: "mlops-end-to-end-cluster"
+          spark_python_task:
+            python_file: "scripts/preprocess.py"
+            parameters: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)